core/src/main/scala/spinal/core/Spinal.scala

```diff
@@ -174,7 +174,8 @@ case class SpinalConfig(mode                           : SpinalMode = null,
                         bitVectorWidthMax              : Int = 4096,
                         var singleTopLevel             : Boolean = true,
                         var noAssertAtTimeZero         : Boolean = false,
-                        var cutLongExpressions         : Boolean = true
+                        var cutLongExpressions         : Boolean = true,
+                        var cleanOutput                : Boolean = false
```

Comment:
cleanOutput as a name isn't explicit enough i think.

Finding a good short name is quite hard for that one XD
hmmm, maybe a parameter which specify the reverse of the current clean output, something like : 
emitFullComponentBindings, meaning, always connect all the pins of a sub component into its parent ?

Reply:
Default bing true